### PR TITLE
Options section for integrations.

### PR DESCRIPTION
**Pull request by GabrielEValenzuela**
## Description
This PR addresses and closes https://github.com/wazuh/wazuh-documentation/issues/5927 . It expands on the integrations examples using the new feature `options`.

With the changes to this PR, the following issues would be fixed:
- https://github.com/wazuh/wazuh/issues/14479

The changes that have been made are as follows:
- Add new tag inside `integrations`, called `options` that supports a custom JSON object to add or modify the different tags of the integrations.
```xml
<options>JSON</options> <!-- Replace with your custom JSON object -->
```
- Add links to the different APIs so that the user has direct access to the fields he can modify.
- Add a quick example to the `ossec.conf` reference.

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

**Review by MarcelKemp — APPROVED** [review]

**Review by MarcelKemp — APPROVED** [review]

**Comment by javimed** [comment]
@GabrielEValenzuela 
- If this is meant for 4.7.0 release, the base for the PR needs to be `4.7.0` and not `master`.
- In _user-manual/reference/ossec-conf/integration.html#configuration-example_, rather than showing `<options>JSON</options> <!-- Replace with your custom JSON object -->` we could show a specific use example, e.g. `<options>{"pretext": "Custom Title"}</options> <!-- Replace with your custom JSON object -->` for a Slack configuration example.

**Comment by GabrielEValenzuela** [comment]
@javimed 

Thanks Javi, the changes have been applied correctly 😃

File: source/user-manual/reference/ossec-conf/integration.rst [hunk 1]
```diff
@@ -151,12 +154,15 @@ Configuration example
...
     <alert_format>json</alert_format>
+    <options>{"pretext": "Custom Title"}</options> <!-- Replace with your custom JSON object -->
   </integration>
 
   <!-- Integration with PagerDuty -->
   <integration>
     <name>pagerduty</name>
     <api_key>API_KEY</api_key> <!-- Replace with your PagerDuty API key -->
+    <options>{"pretext": "Custom title"}</options> <!-- Replace with your custom JSON object -->
+    <alert_format>json</alert_format> <!-- With the new script this is mandatory -->
```
Comment (javimed) [inline source/user-manual/reference/ossec-conf/integration.rst:165]: What new script? maybe some context is needed if commenting like this

**Review by javimed — APPROVED** [review]

File: source/user-manual/reference/ossec-conf/integration.rst (hunk 1 above)
Comment (GabrielEValenzuela) [reply source/user-manual/reference/ossec-conf/integration.rst:165]: Hi @javimed !

I spoke with @MarcelKemp  and the new script is being maintained.

Mainly, the change is based on the fact that pagerduty previously had a `shell` script and now it is in `python`.
For this reason, as most of the python scripts of integrations, the format of the alert that it expects is in JSON and for this reason it is necessary to change this parameter in the configuration.

Here I let you the associate (PR)[https://github.com/wazuh/wazuh/pull/16090] and (issue)[https://github.com/wazuh/wazuh/issues/5833] 😃

**Review by javimed — APPROVED** [review]

**Pull request merged**